projects/sbb-esta/angular-public/autocomplete/src/autocomplete-hint/autocomplete-hint.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="sbb-autocomplete-hint" role="listitem" [id]="id" [ngClass]="_classList">
```

Comment:
This is not a listitem, since it cannot be selected.
Also the div is unnecessary, since this is not going to be extracted like the autocomplete content
Just use `<ng-content></ng-content>` here